spec/features/resgistration_spec.rb

```diff
@@ -14,58 +15,66 @@
       given(:params) { attributes_for :user }
       before { click_button 'commit' }
       scenario { expect(page).to have_content 'Ви успішно зареєстровані.' }
+      scenario('should deliver email') { expect(ActionMailer::Base.deliveries).to include(registration_email) }
     end
 
     context 'with duplicated name' do
       before { create :user, contest: }
-      given(:params) { attributes_for :user }
+      given(:params) { attributes_for :user, email: 'judy.doe@example.com' }
```

Comment:
why do we add email only for this scenario?

Reply:
An email is also sent when we create the first user (line 22). So we need a different email address for the second user to distinguish its email from the first one.